coordinator-service/src/disk-coordinator.ts

```diff
@@ -78,6 +85,34 @@ export class DiskCoordinator implements Coordinator {
         return this.db
     }
 
+    getParameters(): CeremonyParameters {
+        return clonedeep(this.db.parameters)
+    }
+
+    getContributorChunks(contributorId: string): ChunkInfo[] {
```

Comment:
how will contributor client/participant will use these values?  If the participant has already contributed to a chunk, is it useful to return that chunk?  I'm wondering why/if it's useful to return a chunk if `.contributed` evaluates to `true`.

Reply:
It's perhaps enough to have it just return the total number of chunks (this should stay constant, but it's needed for status). In theory a participant should be able to check which chunks it's locking. I'll change it so that a contributor cannot lock a chunk that already includes its contribution. At least after that the participants shouldn't need the chunks where `contibuted` is `true`.